docs/FinalReport.md

```diff
@@ -0,0 +1,86 @@
+# Movie Matcher Project Final Report
+
+## Members
+
+* Josue Alfaro - jja2244
+* Jeremy Castillo - jcc4428
+* Cesar Gonzalez - cg34887
+* John Starich - js68634
+
+## Site
+
+Login
+<http://tomcat.johnstarich.com/login?redirect=%2F>
+
+Register
+<http://tomcat.johnstarich.com/register>
+
+## API Repo
+<https://github.com/JohnStarich/ee461l>
+
+## UI Repo
+<https://github.com/JohnStarich/ee461l-ui>
+
+## Motivation
+
+There's no easy way to find movies groups of people want to watch. Our audience doesn't have a good way to choose movies they'd like to see, especially not with groups with various preferences.
```

Comment:
I feel like this is a little redundant. I like the last sentence more than the first but I don't think we can just delete the first either... Up to you on that one!